src/screens/ProfileScreen/index.js

```diff
@@ -558,12 +561,6 @@ const ProfileScreen = ({route}) => {
     await downVote(post);
   };
 
-  const handleOnEndReached = () => {
-    if (!isLastPage) {
-      getMyFeeds(postOffset);
-    }
-  };
-
   function handleRefresh() {
     setLoading(true);
     setIsLastPage(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleOnEndReached` function has been removed. This could potentially affect the functionality of infinite scrolling. Make sure the new approach with `fetchNextFeeds` covers this functionality and doesn't introduce any regressions.

```diff
- const handleOnEndReached = () => {
-   if (!isLastPage) {
-     getMyFeeds(postOffset);
-   }
- };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->